package.json

```
@@ -31,12 +35,17 @@
   },
   "homepage": "https://github.com/holepunchto/bare-crypto#readme",
   "dependencies": {
-    "bare-stream": "^2.0.0"
+    "bare-stream": "^2.6.3"
   },
   "devDependencies": {
+    "bare-buffer": "^3.0.1",
     "brittle": "^3.5.0",
     "cmake-bare": "^1.1.6",
     "cmake-fetch": "^1.0.0",
-    "standard": "^17.0.0"
+    "prettier": "^3.4.2",
+    "prettier-config-standard": "^7.0.0"
+  },
+  "peerDependencies": {
+    "bare-buffer": "*"
```

```suggestion
    "bare-buffer": "*"
  },
  "peerDependenciesMeta": {
    "bare-buffer": {
      "optional": true
    }
```

To avoid https://github.com/holepunchto/bare-fs/issues/8.